src/IdeaStatiCa.BimImporter/BimImporter.cs

```diff
@@ -84,19 +84,19 @@ public class BimImporter : IBimImporter
 
 		/// <inheritdoc cref="IBimImporter.ImportConnections"/>
 		/// <remarks>Nodes are marked as a connection by following rules:<br/>
-		///  - nodes specified in <see cref="IIdeaModel.GetSelection"/> are connections,<br/>
+		///  - nodes specified in <see cref="IIdeaModel.GetSingleSelection"/> are connections,<br/>
 		///  - nodes with two or more connecting member are connections.
 		/// </remarks>
-		/// <exception cref="InvalidOperationException">Throws if <see cref="IIdeaModel.GetSelection"/> returns null out arguments.</exception>
+		/// <exception cref="InvalidOperationException">Throws if <see cref="IIdeaModel.GetSingleSelection"/> returns null arguments.</exception>
 		public ModelBIM ImportConnections(CountryCode countryCode)
 		{
 			_remoteApp?.SendMessageLocalised(MessageSeverity.Info, LocalisedMessage.ImportingConnections);
 
-			InitImport(out ISet<IIdeaNode> selectedNodes, out ISet<IIdeaMember1D> selectedMembers, out ISet<IIdeaConnectionPoint> connectionPoints);
-			return ProcessSelectedModel(countryCode, selectedNodes, selectedMembers, connectionPoints);
+			var selection = InitBulkImport();
+			return ProcessSelectedModel(countryCode, selection.Nodes, selection.Members, selection.Members2D, selection.ConnectionPoints);
 		}
 
-		private ModelBIM ProcessSelectedModel(CountryCode countryCode, ISet<IIdeaNode> selectedNodes, ISet<IIdeaMember1D> selectedMembers, ISet<IIdeaConnectionPoint> connectionPoints)
+		private ModelBIM ProcessSelectedModel(CountryCode countryCode, ISet<IIdeaNode> selectedNodes, ISet<IIdeaMember1D> selectedMembers, ISet<IIdeaMember2D> selectedMembers2D, ISet<IIdeaConnectionPoint> connectionPoints)
```

Comment:
Možná bychom mohli posílat celý Selection do této funkce?

Reply:
OK, změnil jsem to.